apps/zui/src/views/results-pane/run-results-query.tsx

```diff
@@ -15,7 +15,7 @@ export const runResultsMain = createHandler(
 
     // See if we can paginate this query
     await waitForSelector(QueryInfo.getIsParsed).toReturn(true)
-    const canPaginate = !select(QueryInfo.getIsSummarized)
+    const canPaginate = select(QueryInfo.hasAggregation)
```

Comment:
```suggestion
    const canPaginate = !select(QueryInfo.hasAggregation)
```

Tabling the broader discussion for the moment, I think the loss of the `!` was a mistake here. I just ran a quick test with this branch as of commit abc397c and as this video shows that had us losing paging for search results but still having it for aggregations. Restoring the `!` seems to bring back what we had before.

https://github.com/brimdata/zui/assets/5934157/930f16d5-0c22-4294-91bf-ce56e02b2644